crates/sui-graphql-rpc/src/test_infra/cluster.rs

```diff
@@ -87,12 +93,9 @@ pub async fn serve_executor(
     graphql_connection_config: ConnectionConfig,
     internal_data_source_rpc_port: u16,
     executor: Arc<dyn NodeStateGetter>,
-    env_vars: BTreeMap<String, String>,
+    object_snapshot_min_checkpoint_lag: Option<usize>,
+    object_snapshot_max_checkpoint_lag: Option<usize>,
 ) -> ExecutorCluster {
-    for (k, v) in env_vars {
```

Comment:
I think you may need to set a new base for your PR -- I think you're currently based on the PR you closed recently?